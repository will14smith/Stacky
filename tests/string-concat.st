main () -> () { "abc" "def" concat print }