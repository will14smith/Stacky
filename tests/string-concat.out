=== Output ===
abcdef

=== Exit ===
0
